src/TeamStore.DevUI/wwwroot/js/site.js

```diff
@@ -85,8 +85,57 @@ function bindAllPasswordCopyToClipboard() {
     });
 };
 
+function bindSearchAssetInput() {
+    $("div.dropdown#assetSearchAutocomplete input").on('focusin', function (e) {
+        var getUrl = "/Home/GetAssetResults";
+        var token = $('input[name="__RequestVerificationToken"]').val(); // prepare token
+
+        $.ajax({
+            type: 'GET',
+            url: getUrl,
+            contentType: 'application/json',
+            cache: false,
+            headers: { 'RequestVerificationToken': token },
+            success: function (successResult) {
+                var assetsList = $(this).siblings('ul.dropdown-menu');
+                assetsList.empty();
+                for (var i = 0; i < successResult.length; i++) {
+                    var entry = successResult[i];
+                    assetsList.append('<li class="asset invisible-asset"><a href="/Projects/Details/' + entry.projectId + '">' + entry.title.toLowerCase() + '</a></li>');
+                }
+
+                if ($(this).val().length) {
+                    $(this).triggerHandler('input');
+                }
+            }.bind(this),
+            error: function (XMLHttpRequest, textStatus, errorThrown) {
+                $(this).css("color", "red");
```

Comment:
There could be a more meaningful error handling, I've just forgotten this line here.